pkg/services/ngalert/models/alert_rule.go

```diff
@@ -428,6 +428,11 @@ type AlertRuleGroupKey struct {
 	RuleGroup    string
 }
 
+type AlertRuleGroupWithFolderTitleKey struct {
```

Comment:
Nit: I think you might want to change the naming here, so it follows the same pattern as the other keys.
```suggestion
type AlertRuleGroupKeyWithFolderTitle struct {
```

Reply:
Thanks, makes sense! Updated: https://github.com/grafana/grafana/pull/88289/commits/77e4283de2e3a6c14a44093c13e4fa2df34bab34